examples/edge_bundling.ipynb

```diff
@@ -366,12 +369,10 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "render_points(circular_nodes_df, width=4000, height=4000)"
+    "render_points(circular_nodes_df, width=2000, height=2000)"
```

Comment:
With the reduced width and height, the plotted points are clipped. The original dimensions show the full plot.